cmd/ef_tests/test_runner.rs

```diff
@@ -86,8 +87,13 @@ pub fn build_store_for_test(test: &TestUnit) -> Store {
     let store =
         Store::new("store.db", EngineType::InMemory).expect("Failed to build DB for testing");
     let block_number = test.genesis_block_header.number.as_u64();
+    let hash = H256::random();
     store
-        .add_block_header(block_number, test.genesis_block_header.clone().into())
+        .set_canonical_block_hash(block_number, hash)
```

Comment:
Why are you sending a random hash instead of the correct one? Why is this not breaking anything?

Reply:
Deleted this!